Runtimes/Core/runtime/CMakeLists.txt

```diff
@@ -72,6 +72,9 @@ if(SwiftCore_ENABLE_RUNTIME_LEAK_CHECKER)
     Leaks.mm)
 endif()
 
+check_symbol_exists("dladdr" "dlfcn.h" SwiftCore_HAS_DLADDR)
+check_symbol_exists("dlsym" "dlfcn.h" SwiftCore_HAS_DLSYM)
```

Comment:
I wonder if we should hoist these to the top-level instead

Reply:
Sure, sounds good to me.